### ISSUE TITLE
Warning: Replacing docs 

### ISSUE THREAD

**Issue by GregPlowman** [JuliaIO/FileIO.jl#246]
I'm getting these warnings with `FileIO` `v1.2.0`.
Affects Julia versions `v1.0.5`, `v1.1.1`, `v1.2.0`, `v1.3.0`.

```
┌ Warning: Replacing docs for `FileIO.filename :: Tuple{Any}` in module `FileIO`
└ @ Base.Docs docs\Docs.jl:223
┌ Warning: Replacing docs for `FileIO.file_extension :: Tuple{Any}` in module `FileIO`
└ @ Base.Docs docs\Docs.jl:223
```

**Comment by timholy** [comment]
This is a Julia bug. Fixed in https://github.com/JuliaLang/julia/pull/34125

**Comment by ExpandingMan** [comment]
Is there anything that can be done in FileIO to silence this warning?  This won't be fixed in 1.3.1 so clearly it'll be a while.

**Comment by timholy** [comment]
Someone can delete the `@nospecialize` annotations. I'm not against this but am traveling so won't do it myself.
